learn/guides/graphql-client-tool.md

```diff
@@ -14,69 +14,154 @@ redirect_from:
 
 ```bash
-bal graphql [-i | --input] <graphql-configuration-file-path> [-o | --output] <output-location> 
+bal graphql -i graphql.config.yaml
 ```
 
-This will generate a Ballerina client with remote operations corresponding to each GraphQL query/mutation in the GraphQL document (`.graphql document`). For example,
+This will generate the Ballerina sources (i.e., the three Ballerina files below) from the given GraphQL config.
+
+1. `query_country_client.bal` - contains the Ballerina client with remote operations corresponding to each GraphQL query/mutation in the GraphQL document
+2. `utils.bal` - contains all the utility methods related to the client stub
+3. `types.bal`- contains all the Ballerina data types extracted from the GraphQL schema (SDL)
+
+For the example above, it would generate the client file below.
+
+```ballerina
+import ballerina/http;
+import ballerina/graphql;
+
+public isolated client class QuerycountryClient {
+    final graphql:Client graphqlClient;
+    public isolated function init(string serviceUrl, http:ClientConfiguration clientConfig = {}) returns graphql:ClientError? {
+        graphql:Client clientEp = check new (serviceUrl, clientConfig);
+        self.graphqlClient = clientEp;
+        return;
+    }
+    remote isolated function countryByCode(string code) returns CountryByCodeResponse|graphql:ClientError {
+        string query = string `query countryByCode($code:ID!) {country(code:$code) {name}}`;
+        map<anydata> variables = {"code": code};
+        json graphqlResponse = check self.graphqlClient->executeWithType(query, variables);
+        return <CountryByCodeResponse> check performDataBinding(graphqlResponse, CountryByCodeResponse);
+    }
+}
+```
+
+In general,
 
 ```bash
-bal graphql -i graphql.config1.yaml
+bal graphql [-i | --input] <graphql-configuration-file-path> [-o | --output] <output-location> 
 ```
+
+This will generate a Ballerina client with remote operations corresponding to each GraphQL query/mutation in the GraphQL document (`.graphql document`). The generated sources will be written into the same directory from which the command is executed (i.e., the Ballerina package root directory). For more information, see [GraphQL to Ballerina](/learn/cli-documentation/graphql/#graphql-to-ballerina). 
+
 The above command can be run from anywhere on the execution path. It is not mandatory to run it from within a Ballerina project.
 
-This will generate the files below.
+### Sample usage of the generated client
 
-1. `<document_name>_client.bal` - Ballerina client stub 
-2. `utils.bal` - util file for the relevant `utils` methods related to the client stub
-3. `types.bal`- schema file for the configured GraphQL schema
+The generated client can be used to execute the queries/mutations on the GraphQL API. The example below shows the usage of the generated client in a Ballerina program. Make sure you have copied the generated files into the root of the Ballerina package.
```

Comment:
```suggestion
The generated client can be used to execute the queries/mutations on the GraphQL API. The example below shows the usage of the generated client in a Ballerina program. 

1. Copy the generated files into the root of the Ballerina package.
```
Seems like we need numbered steps for this sequence of instructions. This can be step 1.